odl/space/npy_tensors.py

```diff
@@ -0,0 +1,2024 @@
+﻿# Copyright 2014-2016 The ODL development group
```

Comment:
This whole file is basically a "translated" version of `npy_ntuples.py`, therefore so many lines. The matrix weighting is gone, though.